floodviz/tests/test_hydrograph_utils.py

```diff
@@ -194,6 +169,12 @@ def test_valid_data_missing_count(self):
     def test_valid_data_missing_value(self):
         ret = parse_hydrodata(self.mock_data_missing_points)
         for dp in ret:
-            if dp.get('name') is 'no_data':
-                self.assertEqual(dp.get('value'), 'NA')
+            try:
```

Comment:
The point is you should know where the NA's are. These are tests and you should know exactly what you are expecting.  So yes. We've hardcode the intervals in the function itself so it's reasonable to have to change the tests if those constants are updated.